docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Apache Kafka® MirrorMaker 2 log analysis using the offset sync tool
+sidebar_label: Offset sync log analysis
+---
+
+Analyze key log messages during data synchronization with Apache Kafka® MirrorMaker 2 using Aiven’s offset sync inspection tool.
+This tool tracks the progress of synchronization between source and destination clusters.
+You can run it locally after setting up the required environment to review offset
+syncing between source and target Apache Kafka topics. The tool generates messages
+that indicate partition health and highlight when intervention is needed, making it
+particularly useful during large data migrations.
+
+## Prerequisites
+
+Before you download and run the tool, ensure you have the following:
+
+- Basic understanding of Kafka shell tools.
+- A valid Apache Kafka MirrorMaker 2 configuration file with the necessary source and
+  target cluster connectivity fields.
+
+## Download the offset sync inspection tool
+
+Download and run the **offset sync inspection tool** to analyze offset syncing in
+MirrorMaker 2:
+
+- [Download the offset sync inspection tool](https://github.com/aiven/kafka/releases/tag/mm2-offset-sync-inspector-0.1)
+
+## Run the tool
+
+The offset sync inspection tool outputs logs in CSV format. Use the logs to check the
```

Comment:
I think it's odd to say that the logs are CSV-formatted. It's "output", "data", or a "result", logs are something separate.

Reply:
Thanks. I see what you mean here. I will update this to:
“The offset sync inspection tool outputs data in CSV format. Use this output to check the sync status and detect any issues with the offset syncing process.”